src/ticks/debugger.c

```diff
@@ -228,6 +232,64 @@ void debugger_init()
     exp_engine_init();
 }
 
+uint8_t debugger_read_symbol_file(char* symbol_file) {
+    if (access(symbol_file, F_OK)) {
+        return 1;
+    }
+
+    read_symbol_file(symbol_file);
+
+    char* filename = basename(symbol_file);
```

Comment:
There's zbasename() in common/dirname.c for compatibility purposes - I don't think basename exists for windows.

Reply:
so `src/common` is a separate library, are you suggesting linking it up, or copying those down? Seems like if I do copy, I only need to copy one file.

Reply:
Just add the files to the makefile